js/components/countDownClock.js

```diff
@@ -165,7 +173,15 @@ var CountDownClock = React.createClass({
     OO.log("startUpNext");
     var eventData = {
       "clickedVideo" : this.props.upNextInfo.upNextData,
-      "custom" : {"source": CONSTANTS.SCREEN.UP_NEXT_SCREEN}
+      "custom": {
```

Comment:
If this is the same as the one above in startDiscoveryVideo, can we move this into a separate function to avoid duplicate code?

Reply:
@aeng7 refactored this into a few functions now in Utils